### PR TITLE
Expose span scope Arel

### LINKED ISSUE #23: Expose span Arel
#22 exposes "current" scope

The goal is so that an application using Acts As Span will be able to use Acts As Span's SQL definition of "current" (or "future" or "current or future", etc).

In the above linked PR, an application can get there with

```ruby
SomeClass.current_condition(Date.current)
```

### PR DESCRIPTION
**Pull request by izzergh**
## Checklist
- [x] Continuous integration passes, or no functional code was changed
- [x] Changelog entry added, or no entry is necessary

## Notes
Also adds specs for `expired`, `current_or_future`, and `future`

resolves #23

### CODE REVIEW

File: lib/acts_as_span/span_klass/status.rb [hunk 1]
```diff
@@ -1,59 +1,76 @@
+# frozen_string_Literal: true
+
 require 'active_support'
 
 module ActsAsSpan
   class SpanKlass
+    # Defines class-level methods for relative span scopes.
```
Comment (aeturnumfluctus) [inline lib/acts_as_span/span_klass/status.rb:7]: documentation ftw! :fireworks:

File: lib/acts_as_span/span_klass/status.rb [hunk 2]
```diff
@@ -1,59 +1,76 @@
...
-
-        # returns an Arel node usable within an ActiveRecord `where` clause
+        # The *_condition methods return Arel nodes for composition purposes.
+        #
+        # They accept keyword arguments:
+        #
+        # * query_date: a Date object, the reference date
+        # * table: an Arel table, the table whose columns will be used
+        #
+        # `table` can also be Arel::Nodes::TableAlias
```
Comment (aeturnumfluctus) [inline lib/acts_as_span/span_klass/status.rb:56]: (*Opinon/Question*) What do you think about moving this documentation such that it's included in the class-level documentation, above? It just seems funky to me to document the behavior of multiple methods that return different results in this way.

**Review by aeturnumfluctus — COMMENTED** [review]
This looks good to me. I'd hold out on merging until you Octo's had a chance to weigh in, though.

Great work, @izzergh ! :tada:

File: lib/acts_as_span/span_klass/status.rb (hunk 2 above)
Comment (izzergh) [reply lib/acts_as_span/span_klass/status.rb:56]: That makes sense.

**Review by oneamtu — APPROVED** [review]
Looks g to me, nice work!

**Comment by oneamtu** [comment]
not in scope for this PR, but this can go along well with https://github.com/annkissam/acts_as_span/issues/26 @izzergh - needing those scopes exposed in the API more consistently has come up on LD; they'd make @petrucioata and @katemacewan's lives easier

**Pull request merged**